ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -464,6 +464,9 @@ def __array__(self, dtype=None, /):
         if not dtype:
             return self
         return np_frontend.array(self, dtype=dtype)
+    
+    def __array_wrap__(self, out_arr, context=None):
+        return np_frontend.array(out_arr)
```

Comment:
I think when `context` is None, we should be returning a view of `self`, but not sure what `context` we need to support, if any. @hirwa-nshuti, I think you are the most familiarized with numpy. Could you also review this?

Reply:
Yes ma'am you are right it should return a view of self,

Reply:
I've done some changes, please check once more ma'am